content/zh-cn/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md

```diff
@@ -225,8 +225,8 @@ Here are some of the images we built
 - `gcr.io/kubernetes-e2e-test-images/volume/iscsi:2.0`
 - `gcr.io/kubernetes-e2e-test-images/volume/nfs:1.0`
 - `gcr.io/kubernetes-e2e-test-images/volume/rbd:1.0.1`
-- `k8s.gcr.io/etcd:3.3.15`
-- `k8s.gcr.io/pause:3.1`
+- `registry.k8s.io/etcd:3.3.15` (镜像自发布以来已更改（以前使用的仓库为 "k8s.gcr.io"))
+- `registry.k8s.io/pause:3.1` (镜像自发布以来已更改（以前使用的仓库为 "k8s.gcr.io"))
```

Comment:
I copied this sentence from some other document. Can someone please check if it makes any sense in this context?

Reply:
You are right. This sentence is a translation of English `(image changed since publication - previously used registry "k8s.gcr.io")`
/lgtm
Waiting for #40585 to merge.